### DISCUSSION TITLE
lower the low lvl bg people requirements for start

### DISCUSSION THREAD

**Discussion by Kirishiki** [chromiecraft/chromiecraft#5941] [Suggestions]
could you lower the required amount of people that bg can start at lower lvls so people can actually pvp on those low lvl brackets. its pure torture to try find even 7 people for arathi basin with todays population. if its lowered the bgs would pop more and might attract others to join in when they see announcements of them being started.

**Comment by balleny** [comment]
the amount of players required for arathi basin has already been lowered from 30 to 8.
lowering it further would practically cancel out all the sense of the bg (i would even dare and go as far as saying below 20 people is already bad)

**Reply by Kirishiki** [reply]
id just want to play bgs at low lvl and atm theyre not happening at all even when its call to arms.. managed to get 7/8 yesterday, whispering people to queue it, begging in world chat.. then some of those joined dungeon instead and that was it for that day..
